ruleset/rules/0690-gcp_rules.xml

```diff
@@ -283,6 +283,245 @@ ID: 65000 - 65499
+    </rule>
+
+    <!-- collection gcp_pub_sub topic creation -->
+    <rule id="65052" level="3">
+        <if_sid>65000</if_sid>
+        <field name="gcp.protoPayload.methodName" type="pcre2">^google\.pubsub\.v\d*\.Publisher\.CreateTopic$</field>
+        <field name="gcp.protoPayload.authorizationInfo" type="pcre2">"granted":\strue</field>
+        <description>GCP pub/sub topic created</description>
+        <mitre>
+            <id>T1530</id>
+        </mitre>
+    </rule>
+
+    <!-- defense evasion gcp_firewall_rule created -->
+    <rule id="65053" level="3">
+        <if_sid>65000</if_sid>
+        <field name="gcp.protoPayload.methodName" type="pcre2">^v\d*\.compute\.firewalls\.insert$</field>
+        <description>GCP firewall rule created</description>
+        <mitre>
+            <id>T1562</id>
+        </mitre>
+    </rule>
+
+    <!-- defense evasion gcp_firewall_rule deleted -->
+    <rule id="65054" level="3">
+        <if_sid>65000</if_sid>
+        <field name="gcp.protoPayload.methodName" type="pcre2">^v\d*\.compute\.firewalls\.delete$</field>
+        <description>GCP firewall rule deleted</description>
+        <mitre>
+            <id>T1562</id>
+        </mitre>
+    </rule>
+
+    <!-- defense evasion gcp_firewall_rule modified -->
+    <rule id="65055" level="3">
+        <if_sid>65000</if_sid>
+        <field name="gcp.protoPayload.methodName" type="pcre2">^v\d*\.compute\.firewalls\.patch$</field>
+        <description>GCP firewall rule modified</description>
+        <mitre>
+            <id>T1562</id>
+        </mitre>
+    </rule>
+
+    <!-- defense evasion gcp_logging_bucket deletion -->
+    <rule id="65056" level="3">
+        <if_sid>65000</if_sid>
+        <field name="gcp.protoPayload.methodName" type="pcre2">^google\.logging\.v\d*\.ConfigServiceV\d*\.DeleteBucket$</field>
+        <field name="gcp.protoPayload.authorizationInfo" type="pcre2">"granted":\strue</field>
+        <description>GCP logging bucket deleted</description>
+        <mitre>
+            <id>T1562</id>
+        </mitre>
+    </rule>
+
+    <!-- defense evasion gcp_logging_sink deletion -->
+    <rule id="65057" level="3">
+        <if_sid>65000</if_sid>
+        <field name="gcp.protoPayload.methodName" type="pcre2">^google\.logging\.v\d*\.ConfigServiceV\*\.DeleteSink$</field>
+        <field name="gcp.protoPayload.authorizationInfo" type="pcre2">"granted":\strue</field>
+        <description>GCP logging sink deleted</description>
+        <mitre>
+            <id>T1562</id>
+        </mitre>
+    </rule>
+
+    <!-- defense evasion gcp_pub_sub_subscription deletion -->
+    <rule id="65058" level="3">
+        <if_sid>65000</if_sid>
+        <field name="gcp.protoPayload.methodName" type="pcre2">^google\.pubsub\.v\d*\.Subscriber\.DeleteSubscription$</field>
+        <field name="gcp.protoPayload.authorizationInfo" type="pcre2">"granted":\strue</field>
+        <description>GCP pub/sub subscription deleted</description>
```

Comment:
Same as before, I'm not fully understanding the description

Reply:
An adversary may delete a subscription to a topic in order to negate access to the logs being sent to said topic. pub/sub is a mechanism offered by GCP to send logs following a publisher/subscriber methodology.